--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -60,6 +60,3 @@
 
       - name: Run unit tests
         run: poetry run poe tests
-
-      - name: Run man page generator
-        run: poetry run poe doc
--- a/.gitignore
+++ b/.gitignore
@@ -5,6 +5,7 @@
 /debian/itchcraft/
 /debian/itchcraft.*.debhelper
 /debian/itchcraft.substvars
+/debian/*.rst
 /dist/
 /.pc/
 /.venv/
--- a/doc/sphinx/conf.py
+++ b/doc/sphinx/conf.py
@@ -14,7 +14,6 @@
 # https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration
 
 extensions = [
-    'myst_parser',
     'sphinx.ext.autodoc',
     'sphinx.ext.napoleon',
 ]
--- a/doc/sphinx/index.rst
+++ b/doc/sphinx/index.rst
@@ -5,5 +5,4 @@
    :maxdepth: 2
    :caption: Contents:
 
-.. include:: ../../USAGE.md
-   :parser: myst_parser.sphinx_
+.. include:: ../../debian/USAGE.rst
